docs/README.md

```diff
@@ -58,8 +58,53 @@ Installation process
     * OpenShift
         ```
         helm install csi-baremetal csi/csi-baremetal-deployment --devel --set scheduler.patcher.enable=true \
-      --set platform=openshift --set global.registry=$REGISTRY --set global.registrySecret=$DOCKER_REGISTRY_SECRET
+      --set platform=openshift --set global.registry=$REGISTRY --set global.registrySecret=$DOCKER_REGISTRY_SECRET \
+      -n $NAMESPACE
         ```
+      **Note:** In case of CSI deployment at Openshift platform in non default namespace, CSI ServiceAccounts \
+      csi-node-sa and csi-baremetal-extender-sa will require Privileged SCC. In case there are no such permissions \
+      CSI Operator will not deploy node and scheduler extender daemonsets. You can see the following events in such case:
+      ```
+      65m         Warning   NodeRoleValidationFailed       deployment/csi-baremetal                              ServiceAccount csi-node-sa has insufficient securityContextConstraints, should have privileged
+      65m         Warning   ExtenderRoleValidationFailed   deployment/csi-baremetal                              ServiceAccount csi-baremetal-extender-sa has insufficient securityContextConstraints, should have privileged
+      ```
+      To make them deployable - create the following role and rolebinding in deployed namespace:
```

Comment:
Can we mention `oc adm policy add-scc-to-user privileged -z ...` commands here instead?

Reply:
By this command two resources are used:
1. ClusterRole system:openshift:scc:privileged;
2. RoleBinding system:openshift:scc:privileged.
So in this case user give access to mutual ClusterRole instead of Role. We made it in more restricted manner  just using the Role instead of ClusterRole.